src/app/pages/flex-page/blocks/CardsBlock.scss

```diff
@@ -0,0 +1,35 @@
+@import 'pattern-library/core/pattern-library/headers';
+
+.flex-page.page div.content-block-cards {
+    display: grid;
+    margin: 0 auto;
+
+    /* needs responsive design */
+    grid-template-columns: repeat(auto-fit, calc(var(--card-size, 27) * 1rem));
+    grid-gap: ($normal-margin * 2);
+    justify-content: center;
+
+    > .content-block-card {
+      padding: $normal-margin;
+
+      /* to show above divider images */
+      position: relative;
+      z-index: 1;
+    }
+
+    &.card_style_rounded > .content-block-card {
+      border-radius: 1rem;
+      border: thin solid ui-color(black);
+
+      /* is this the right color for default background? */
+      background-color: #fff;
```

Comment:
Pattern library colors are defined here:
https://github.com/openstax/pattern-library/blob/master/core/pattern-library/colors.scss
You could use `ui-color(default)` or `ui-color(white)`